sncrs/data/admin.py

```diff
@@ -23,6 +23,9 @@ class AliasInline(admin.TabularInline):
 class GreetingInline(admin.TabularInline):
     model = Greeting
 
+class WhineInline(admin.TabularInline):
```

Comment:
Inline admin interfaces are great for editing a model from a related model! In this case, if the goal is to be able to add Whines from the Person admin interface, you need to add `WhineInline` to the list from line 33-37
Alternatively, if you don't intended for that to be the case, I would suggest removing lines 26-28

Reply:
Ahhhh OK, yeah I had forgotten what the *InLines were used for. Thanks, I added it to PersonAdmin